src/main/resources/config/application.yaml

```diff
@@ -18,4 +18,4 @@ powsybl-ws:
     name: filters
 
 server:
-  max-http-header-size: 64000
+  max-http-request-header-size: 64000
```

Comment:
do we need http responses max header sizes ?

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide

> 'server.max-http-header-size'
Previously, the server.max-http-header-size was treated inconsistently across the four supported embedded web servers. When using Jetty, Netty, or Undertow it would configure the max HTTP request header size. When using Tomcat it would configure the max HTTP request and response header sizes.

> To address this inconsistency, server.max-http-header-size has been deprecated and a replacement, server.max-http-request-header-size, has been introduced. Both properties now only apply to the request header size, irrespective of the underlying web server.

> To limit the max header size of an HTTP response on Tomcat or Jetty (the only two servers that support such a setting), use a WebServerFactoryCustomizer.


If yes, a new property is coming in 3.2 server.tomcat.max-http-response-header-size (also jetty but we use tomcat) https://github.com/spring-projects/spring-boot/commit/93d46d11e91d21ccbe2e5b451c051e9efdc57b2d

Reply:
I don't know if we need it, I just migrated to what seemed like the equivalent of the deprecated property